atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -727,9 +733,9 @@ class KojiImportSourceContainerPlugin(KojiImportBase):
     key = PLUGIN_KOJI_IMPORT_SOURCE_CONTAINER_PLUGIN_KEY  # type: ignore
 
     def __init__(self, workflow, blocksize=None,
-                 target=None, poll_interval=5):
+                 target=None, poll_interval=5, userdata=None):
         super(KojiImportSourceContainerPlugin, self).__init__(workflow, blocksize,
-                                                              target, poll_interval)
+                                                              target, poll_interval, userdata)
```

Comment:
The inits in both subclasses could be removed

Reply:
done